src/features/Apiexplorer/SubscribeRenderer/index.tsx

```diff
@@ -30,10 +30,16 @@ function SubscribeRenderer<T extends TSocketSubscribableEndpointNames>({
   const [toggle_modal, setToggleModal] = useState(false);
   const [is_not_valid, setIsNotValid] = useState(false);
 
+  const subscribe_ref: any = useRef();
+
   useEffect(() => {
     if (error && error.code === 'AuthorizationRequired') {
       setToggleModal(true);
     }
+
+    return () => {
+      if (subscribe_ref.current) subscribe_ref.current.unsubscribe();
+    };
   }, [error]);
```

Comment:
subscribe_ref should be in the dependency array.

Reply:
If I declare it in the dependancy array streaming will not happen